docs/design/features/byreflike-generics.md

```diff
@@ -24,6 +24,8 @@ The following instructions are already set up to support this feature since thei
 - `isinst` &ndash; Will always place `null` on stack.
 - `castclass` &ndash; Will always throw `InvalidCastException`.
 
+**NOTE** There are sequences involving some of the above instructions that are considered optimizations. These sequences represent cases that will remain valid regardless of a `T` being ByRefLike&mdash;see ["Special IL Sequences" section](#special_il_sequences) below for details.
```

Comment:
```suggestion
**NOTE** There are sequences involving some of the above instructions that will remain valid regardless of a `T` being ByRefLike&mdash;see ["Special IL Sequences" section](#special_il_sequences) below for details.
```
I would not call these special sequences "optimizations". I see optimization as an optional operation that may or may not happen. It is not the case here. The special sequences must be recognized for the program to work.

Reply:
Agree.